web_timeline_calendar/models/calendar_event.py

```diff
@@ -0,0 +1,33 @@
+# Copyright 2019 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import models, fields, api
+
+
+class CalendarEvent(models.Model):
+
+    _inherit = "calendar.event"
+
+    user_ids = fields.Many2many(comodel_name='res.users',
+                                relation='calendar_event_res_users_rel',
+                                column1='calendar_event_id',
+                                column2='res_users_id',
+                                compute='_compute_user_ids',
+                                string="Users",
+                                store=True)
```

Comment:
Why is it necessary to have this field store=True?

Reply:
For performance. The computation is really heavy (at least in the customer env) so this way the information is coming from the DB instead of the computation at the beginning.